### PR TITLE
[MIG][11.0] partner_bank_active

### PR DESCRIPTION
**Pull request by Kiplangatdan**

### CODE REVIEW

File: partner_bank_active/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Copyright 2017-2019 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
```
Comment (NL66278) [inline partner_bank_active/__manifest__.py]: Please change license files to https://

**Review by NL66278 — APPROVED** [review]
:+1: LGTM Thanks for the migration!

**Review by rafaelbn — APPROVED** [review]

**Pull request merged**